src/main/lrsql/system/lrs.clj

```diff
@@ -28,7 +28,8 @@
             [lrsql.util.auth      :as auth-util]
             [lrsql.util.statement :as stmt-util]
             [lrsql.init.authority   :refer [make-authority-fn]]
-            [lrsql.util.concurrency :refer [with-rerunable-txn]])
+            [lrsql.util.concurrency :refer [with-rerunable-txn]]
+            [lrsql.util.oidc :as oidc])
```

Comment:
I wonder how much of the stuff in `lrsql.util.oidc` can be split off into a `lrsql.init.oidc` namespace.

Reply:
I tried to keep the footprint as small as possible. What would go there, the authority function stuff?

Reply:
Yes, as per my other comment. And I don't think two namespaces is much more of a concern than one, especially when you have this clear divide between "init" and "runtime".